streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamTransformTest.java

```diff
@@ -71,7 +72,8 @@ public void init(final ProcessorContext context) {
                 }
 
                 @Override
-                public void close() {}
+                public void close() {
+                }
```

Comment:
Why?

Reply:
i don't know. As i do auto stylecheck format using intellij.  using the stylecheck provided in kafka folder.